README.md

```diff
@@ -27,17 +27,18 @@ The `initIAdvizeIframe` comes with 2 arguments :
 - `iAdvizePlatform` : the iadvize platform (default: ha).
 
 ```js
-import { initIAdvizeIframe } from '@iadvize-oss/sandboxed-tag';
+import { initIAdvizeIframe } from '@iadvize-oss/boxed-tag';
 
 initIAdvizeIframe(<sid>, <iAdvizePlatform>);
 ```
 
-## Add a sandboxed iframe
+## Add a boxed iframe
 
 Add a sanboxed iframe on your site's main page.
 
 ```html
 <iframe
+  title="iAdvize Boxed tag iframe"
```

Comment:
```suggestion
  title="iAdvize Boxed tag iframe"
```

We should use a comprehensible title, for accessibility
I think we use "iAdvize chat notification frame" for the standard livechat
We should maybe use the same title